index.js

```diff
@@ -0,0 +1,47 @@
+/* 
+page loads with defaults
+listen to the submit event on the form element
+when the submit event happens, we'll do the following...
+
+1. find the text box and extract the text the person typed in there
+2. compare what the user typed in to nyc or new york or new york city
+3. if there's a match then we need to remove any classes on the body tag
+4. and then add the nyc class to to the body tag.
+5. changes the background image to corresponding city
+6. 
+7.
+
+*/
+
+$(document).ready(function() {
+    $("form").submit(function(event) {
```

Comment:
The form is not where the event happens. It happens on the button that they either click or submit. So your code should have been:

`$('#submit-btn').click(function(){`

or

`$('#submit-btn').submit(function(){`